serialization/Schemas/spdx-v3.json

```diff
@@ -0,0 +1,1061 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://spdx.org/spdx/v3/document",
+  "title": "SPDX v3",
+  "version": "spdx-3-model 2022-09-20",
+  "description": "This information model defines the serialized structure of SPDXv3 logical model types",
+  "license": "CC0-1.0",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "payload": {
+      "$ref": "#/definitions/Payload"
+    },
+    "elements": {
+      "$ref": "#/definitions/Elements"
```

Comment:
As payload already has a list of all elements, what is the idea of the "elements" list parallel to "payload"?

Reply:
Elements is an array of uncompressed element values.
Payload is the structure of context information plus the "element" property of compressed element values.

It would probably be clearer to have two different schemas.   Thanks.